gensim/utils.py

```diff
@@ -1168,22 +1175,28 @@ def check_output(stdout=subprocess.PIPE, *popenargs, **kwargs):
     Python 2.6.2
     Added extra KeyboardInterrupt handling
     """
-    try:
-        logger.debug("COMMAND: %s %s", popenargs, kwargs)
-        process = subprocess.Popen(stdout=stdout, *popenargs, **kwargs)
-        output, unused_err = process.communicate()
-        retcode = process.poll()
-        if retcode:
-            cmd = kwargs.get("args")
-            if cmd is None:
-                cmd = popenargs[0]
-            error = subprocess.CalledProcessError(retcode, cmd)
-            error.output = output
-            raise error
-        return output
-    except KeyboardInterrupt:
-        process.terminate()
-        raise
+    logger.debug("COMMAND: %s %s", popenargs, kwargs)
+    command = kwargs.get('args')[0]
+    usr_bin_alias = '/usr/bin/' + command
```

Comment:
What's about windows case too?

Reply:
You've got a real problem here in 1173 row. If popenargs variable would not be empty then python will give an error about positional variable goes after named variable.